scripts/pagespeed_libraries_generator.sh

```diff
@@ -27,9 +27,9 @@
 # Author: vid@zippykid.com (Vid Luther)
 #         jefftk@google.com (Jeff Kaufman)
 
-URL="https://github.com/pagespeed/mod_pagespeed/raw/master/"
-URL+="net/instaweb/genfiles/conf/pagespeed_libraries.conf"
-curl -L -s -S "$URL" \
+URL="https://raw.githubusercontent.com/pagespeed/mod_pagespeed/master/"
+URL2="net/instaweb/genfiles/conf/pagespeed_libraries.conf"
```

Comment:
That's already made and it's not working, that's why I used URL2. Doing what you say make us returning to the original code which is not working. I'm out of idea here...I just found a easy solution to the problem.